server/initializers/session.js

```diff
@@ -16,7 +16,18 @@ module.exports = upgradeInitializer('ah17', {
           if (error) {
             return callback(error)
           } else if (data) {
-            return callback(null, JSON.parse(data))
+            const sessionData = JSON.parse(data)
```

Comment:
old bug, but better fix now - if parsing crashes, callback will not be called, wrap in try/catch